### PR TITLE
Add privacy considerations section

### PR DESCRIPTION
**Pull request by avadacatavra**

### CODE REVIEW

File: explainer.md [hunk 1]
```diff
@@ -168,6 +168,14 @@ function checkOrientation(tipOrientation, metacarpalOrientation) {
 }
 ```
 
+## Privacy and Security Considerations
+
+Exposing hand input poses a risk to both a user's identity and their sensitive characteristics. This API exposes details of hand, finger, and joint size, which could be used to uniquely identify a user. Beyond exposing details of hand size, this API is intended to enable gesture recognition. Gestures have been used in authentication mechanisms as a behavioral biometric, and are likely to expose uniquely identifying information about users.
+
+Medical conditions that produce tremors in the hands may be exposed by this API, as well as whether a user is missing fingers or parts of fingers. Additionally, hand size can reveal age, and possibly gender. Finally, there's the possibility that someone might misuse this API in an attempt to infer ethnic background from hand size, which amounts to hand-based phrenology.
+
+This API can also potentially capture sensitive inputs, e.g. capturing typed input.
```
Comment (Manishearth) [inline explainer.md:177]: This one we can fix by ensuring that hand spaces are also not active during security-blurred states

File: explainer.md [hunk 2]
```diff
@@ -168,6 +168,14 @@ function checkOrientation(tipOrientation, metacarpalOrientation) {
 }
 ```
 
+## Privacy and Security Considerations
+
+Exposing hand input poses a risk to both a user's identity and their sensitive characteristics. This API exposes details of hand, finger, and joint size, which could be used to uniquely identify a user. Beyond exposing details of hand size, this API is intended to enable gesture recognition. Gestures have been used in authentication mechanisms as a behavioral biometric, and are likely to expose uniquely identifying information about users.
+
+Medical conditions that produce tremors in the hands may be exposed by this API, as well as whether a user is missing fingers or parts of fingers. Additionally, hand size can reveal age, and possibly gender. Finally, there's the possibility that someone might misuse this API in an attempt to infer ethnic background from hand size, which amounts to hand-based phrenology.
```
Comment (cabanier) [inline explainer.md:175]: Since all joints have to be returned, missing fingers should not be a concern.

Comment (cabanier) [reply explainer.md:175]: > there's the possibility that someone might misuse this API in an attempt to infer ethnic background from hand size, which amounts to hand-based phrenology

Does the current language in the spec suffice here?

Comment (cabanier) [reply explainer.md:175]: > Medical conditions that produce tremors in the hands may be exposed by this API

This API likely exposes less "tremors" than what controllers would return. Is it worth it to list it as a concern?

**Comment by Manishearth** [comment]
We should add additional concerns in separate PRs, to get this in at least.

**Pull request merged**